Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -317,31 +319,31 @@ public class HTTPClient {
         ///  - `305: Use Proxy`
         ///  - `307: Temporary Redirect`
         ///  - `308: Permanent Redirect`
-        public var followRedirects: Bool
+        public var followRedirects: FollowRedirects
         /// Default client timeout, defaults to no timeouts.
         public var timeout: Timeout
         /// Upstream proxy, defaults to no proxy.
         public var proxy: Proxy?
         /// Ignore TLS unclean shutdown error, defaults to `false`.
         public var ignoreUncleanSSLShutdown: Bool
 
-        public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: Bool = false, timeout: Timeout = Timeout(), proxy: Proxy? = nil) {
+        public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: FollowRedirects = .disabled, timeout: Timeout = Timeout(), proxy: Proxy? = nil) {
```

Comment:
should following redirect be default?

Reply:
I think in server-side setting it makes sense to have defaults to be predictable, and in case of redirects we have potentially unexpected behaviour, but it's not a strong opinion. @Lukasa @weissi what do you think?

Reply:
I would usually err on the side of 'just works' and make it redirect with a limit of 5 or something.

Regarding the parameter: `followRedirects = .disabled` doesn't read as good as `followRedirects: .no`, right?

Reply:
Agreed with @weissi

Reply:
following redirects is now default with loop detection